lib/checks.js

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) 2012 Mathieu Turcotte
  * Licensed under the MIT license.
  */
+'use strict';
 
 var util = require('util');
 
 var errors = module.exports = require('./errors');
 
 function failCheck(ExceptionConstructor, callee, messageFormat, formatArgs) {
+    /* jshint validthis: true */
```

Comment:
jshint warns if you reference `this` (see line 14) in a function that might not be attached to an object.